sdk/deviceupdate/azure-iot-deviceupdate/src/samples/java/com/azure/iot/deviceupdate/ListDevicesSample.java

```diff
@@ -0,0 +1,25 @@
+package com.azure.iot.deviceupdate;
```

Comment:
This is missing copyright comment. Is this manually added?

Reply:
Yes, this one is manual sample. Please fix together with the other one @haolingdong-msft

Reply:
updated